docs/src/install/index.md

```diff
@@ -221,3 +221,42 @@ The packages can be viewed directly from the [Cloudsmith.io](https://cloudsmith.
     </td>
 </tr>
 </table>
+
+## Shell completions
+To make using the `tedge` CLI easier, as of TODO VERSION, `tedge` provides
+tab-completion support. This can be enabled by adding a small configuration to
+your shell environment:
+
+```sh title="~/.bashrc" tab={"label":"bash"}
+eval "$(tedge completions bash)"
```

Comment:
Any specific reason for using eval instead of source?

I would prefer using source using:

```suggestion
source <(tedge completions bash)
```

Reply:
Not particularly, I've changed it now